recipes/lcov/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.13" %}
+
+package:
+  name: lcov
+  version: {{ version }}
+
+source:
+  - url: https://github.com/linux-test-project/lcov/archive/v{{ version }}.tar.gz
+    fn: lcov-{{ version }}.tar.gz
+    sha256: 3650ad22773c56aaf8c5288e068dd35bd03f57659b6455dc6f8e21451c83b5e8
+    patches:
+      - fix_perl_in_conda.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl
+    - make
+  run:
+    - perl
+
+test:
+  commands:
+    - lcov --help
+
+about:
+  home: http://ltp.sourceforge.net/test/coverage/lcov.php
+  summary: 'LCOV is an extension of GCOV, a GNU tool which provides information about
+  what parts of a program are actually executed (i.e. "covered") while running
+  a particular test case.'
+  license: GPL-2
+  license_family: GPL-2
+  license_url: https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+  license_file: '{{ environ["SRC_DIR"] }}/COPYING'
```

Comment:
The jinja2 ref to the source dir is not needed here but clearly does not hurt!

Reply:
you mean I can just add `license_file: COPYING`?

Reply:
Yep!

Reply:
Yep!